Audio cleanups

johnnovak opened the pull request:
# Description

Splitting out cleanup & reformat changes from my upcoming [ESFM PR](https://github.com/dosbox-staging/dosbox-staging/issues/3608) to reduce the noise.

These commits are non-functional cleanups only, most of them are only rename jobs.


# Manual testing

Stuff still compiles, things still work 😎 


# Checklist

_Please tick the items as you have addressed them. Don't remove items; leave the ones that are not applicable unchecked._

I have:

- [x] followed the project's [contributing guidelines](https://github.com/dosbox-staging/dosbox-staging/blob/master/CONTRIBUTING.md) and [code of conduct](https://github.com/dosbox-staging/dosbox-staging/blob/master/CODE_OF_CONDUCT.md).
- [x] performed a self-review of my code.
- [ ] commented on the particularly hard-to-understand areas of my code.
- [x] split my work into well-defined, bisectable commits, and I [named my commits well](https://github.com/dosbox-staging/dosbox-staging/blob/main/CONTRIBUTING.md#commit-messages).
- [x] applied the appropriate labels (bug, enhancement, refactoring, documentation, etc.)
- [ ] [checked](https://github.com/dosbox-staging/dosbox-staging/blob/main/scripts/compile_commits.sh) that all my commits can be built.
- [ ] confirmed that my code does not cause performance regressions (e.g., by running the Quake benchmark).
- [ ] added unit tests where applicable to prove the correctness of my code and to avoid future regressions.
- [ ] made corresponding changes to the documentation or the website according to the [documentation guidelines](https://github.com/dosbox-staging/dosbox-staging/blob/main/DOCUMENTATION.md).
- [ ] [locally verified](https://github.com/dosbox-staging/dosbox-staging/blob/main/DOCUMENTATION.md#previewing-documentation-changes-locally) my website or documentation changes.

japsmits reviewed (changes requested):
Your find/replace buttons must be worn out by now. 😉 Nice work!

Couple of smaller comments, nothing too serious.

johnnovak commented on include/envelope.h:
```diff
@@ -66,42 +66,47 @@ class Envelope {
 	Envelope(const char* name);
 
-	void Process(const bool is_stereo, AudioFrame &frame);
+	void Process(bool is_stereo, AudioFrame& frame);
```
I've recently learned this, and it makes sense to me:

For primitive types, the `const`ness of a function argument is an implementation detail. As this is not a pointer or a reference to some data that can be modified by the caller, from the point of view of the interface, it does not matter whether the implementation allows the primitive type to be changed inside the function or not. So it unnecessarily clutters the interface; it's not a significant detail.

For pointers and references of course it is _very_ significant whether the data the point to can be modified by the function or not; I don't even think the compiler will allow you to drop the const-ness from the declaration because it's significant.

See these more detailed discussions here:

https://stackoverflow.com/a/706071
https://stackoverflow.com/a/15297695

japsmits replied:
I agree that const in the case of by-value passing of parameters is only significant for the implementation, so language-wise there's a point.

However, consistency between definition and implementation is also important. A well aimed find (and replace) for example becomes a tad more difficult if const is in one place, but not in the other. Although it is not important to the caller, is might be important to the developer.

johnnovak replied:
Ok, had a brief chat with the guys, and it's the least surprising to just keep them in sync. It's a very minor thing and arguably it does not really matter either way, so let's just prefer consistency then like you said.

johnnovak commented:
> Your find/replace buttons must be worn out by now. 😉 Nice work!
> 
> Couple of smaller comments, nothing too serious.

Thanks for the quick and thorough review @japsmits! 🎉  Addressed the comments.
